services/httpd/service.go

```diff
@@ -2,38 +2,64 @@ package httpd
 
 import (
 	"crypto/tls"
+	"expvar"
 	"fmt"
 	"log"
 	"net"
 	"net/http"
 	"os"
 	"strings"
+	"sync"
+
+	"github.com/influxdb/influxdb/monitor"
+)
+
+// statistics gathered by the httpd package.
+const (
+	statRequests           = "requests"
+	statCQServed           = "cq_served"
+	statQueriesServed      = "queries_served"
+	statWriteServed        = "write_served"
+	statPingServed         = "ping_served"
+	statWriteBytesReceived = "write_bytes_rx"
```

Comment:
@jwilder - how about `write_req_bytes_rx`? It's the number of bytes POSTed in the body of write requests.